src/standalone/files.go

```diff
@@ -24,7 +24,10 @@ k8s_basepodname %s
 k8s_namespace %s
 `
 
-	k8NodeNameFormatString = `k8s_node_name %s
+	k8HostInfoFormatString = `k8s_node_name %s
+[host]
+tenant
```

Comment:
```suggestion
tenant %s
```
should also be adjusted in other locations

Reply:
fixed in https://github.com/Dynatrace/dynatrace-operator/pull/2151/commits/b4add95502ea42ed738533dcbf9c97b686be4571